src/js/controllers/confirm.js

```diff
@@ -518,11 +571,176 @@ angular.module('copayApp.controllers').controller('confirmController', function(
           cardClaimCode: $scope.amazonGiftCard ? $scope.amazonGiftCard.claimCode : null
         });
       });
+    } else if (fromGlidera) {
+      $ionicHistory.nextViewOptions({
+        disableAnimate: true,
+        historyRoot: true
+      });
+      $ionicHistory.clearHistory();
+      $state.go('tabs.home').then(function() {
+        $state.transitionTo('tabs.buyandsell.glidera');
+      });
     } else {
       $state.go('tabs.send');
     }
   };
 
+  $scope.get2faCode = function(cb) {
+    ongoingProcess.set('Sending 2FA code...', true);
```

Comment:
This text should be added to _ongoingProcess service_ (change this for all calls)